### ISSUE TITLE
Feat: home page with add category functionality

### ISSUE THREAD

**Issue by annabauza** [anitab-org/stem-diverse-tv-cms#12]
### Is your feature request related to a problem? Please describe.
As a user I want to add category into our [backend](https://stem-diverse-tv.herokuapp.com/).

### Describe the solution you'd like
Follow the [design](https://www.figma.com/file/klxnTZJ8Z2uAtxHS1CRTWO/Untitled?node-id=12%3A1) and use add category endpoint from [backend](https://stem-diverse-tv.herokuapp.com/).
![Home (2)](https://user-images.githubusercontent.com/31966073/106385472-668c1e80-63c8-11eb-95ea-51b43923e6fc.png)
![Home (3)](https://user-images.githubusercontent.com/31966073/106385471-64c25b00-63c8-11eb-8b60-0fd8b24bd3f0.png)

### Additional context
This is a home page with Title Categories and + button and popup
ATM there is no get all categories so list of existing categories can not be displayed

**Comment by NenadPantelic** [comment]
Can I take this one @annabauza?

**Comment by annabauza** [comment]
@NenadPantelic you have like 3 or 4 issues. Release issues so we know you work only on 1 at the time.

**Comment by NenadPantelic** [comment]
Sure, I'm nearly done with 1 of them - I will open a PR tomorrow probably. I didn't start with the second one (https://github.com/anitab-org/stem-diverse-tv/issues/24), because I was not sure should we restructure the database or we will solve that on the client side (we discussed that yesterday, so I'm sure now), I will be working on that since tomorrow, but yeah, I can release that until I start working on it. I can finish the one about CONTRIBUTION guidelines today.

**Comment by NenadPantelic** [comment]
I suppose I can work on two issues if they're from different repos?

**Comment by annabauza** [comment]
@isabelcosta ☝️ you think?

**Comment by isabelcosta** [comment]
I think if it is on different repositories its fine. I hope in the open pull requests, you are done with all of your changes and only waiting for review. @NenadPantelic 
cc @annabauza

**Comment by jalajcodes** [comment]
@annabauza  If it's available I'd like to work on it

**Comment by jalajcodes** [comment]
Is this issue available? @annabauza @isabelcosta

**Comment by annabauza** [comment]
yes @jalajcodes and we already have merged get all categories so the list can be implemented. Although login has not been merged yet.

**Comment by jalajcodes** [comment]
@annabauza If you assign me to Login issue I can work on it first.

**Comment by annabauza** [comment]
Use backend swagger to get token. It's not a blocker.

**Comment by annabauza** [comment]
Ok so this one is free I assigned you to login @jalajcodes

**Comment by annabauza** [comment]
> I think if it is on different repositories its fine. I hope in the open pull requests, you are done with all of your changes and only waiting for review. @NenadPantelic
> cc @annabauza

Ty ty

**Comment by NenadPantelic** [comment]
@annabauza Can I take this one if it's available?

**Comment by annabauza** [comment]
@NenadPantelic yes please.

**Comment by NenadPantelic** [comment]
I do, but I cannot fully implement it until we implement some things on the backend (error cases)

**Comment by jalajcodes** [comment]
> I do, but I cannot fully implement it until we implement some things on the backend (error cases)

Gotcha! 😉

**Comment by jalajcodes** [comment]
@NenadPantelic Are you still working on it? we can make it available for OSH

**Comment by NenadPantelic** [comment]
@jalajcodes I unassigned myself, I waited for the things to be solved on the backend. I'm overstacked with some other things, so I made it available for the OSH.

**Comment by annabauza** [comment]
Thanks @NenadPantelic
